src/app/modules/filters/filters-popover/filters-popover.component.scss

```diff
@@ -9,55 +9,58 @@
     }
 
     .popup-body {
-      padding-left: 4em;
+      padding-left: 4em !important;
     }
   }
 }
 
 .popup-container {
-  width: 0em;
   position: absolute;
-  height: 23em;
-  z-index: -1;
+  border-width: 0px;
+  border-style: solid;
+  width: 0em;
+  height: 22.5rem;
+  z-index: 5;
   top: 0;
-  right: 27.5em;
-  background-color: #323232;
+  right: 24rem;
   box-shadow: 0.4rem 0.4rem 0.8rem 0rem #000000;
   transition-property: width;
-  transition-duration: .3s;
+  transition-duration: 0.3s;
 
   .popup-body {
     position: relative;
-    padding: 1em;
     opacity: 0;
     transition-property: opacity;
     transition-duration: 0s;
     transition-delay: 0s;
+
+    ccf-filters-content {
```

Comment:
This should be moved to the ccf-filters-content scss instead of using `!important`